client/src/components/routes/ProtectedRoute.js

```diff
@@ -0,0 +1,27 @@
+// TODO: Add PropTypes
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import auth from '../../auth/auth';
+
+function ProtectedRoute({ component: Component, ...rest }) {
+  return (
+    <Route
+      {...rest}
+      render={props => (auth.isAuthenticated() === 'true' ? (
```

Comment:
isAuthenticated can return a boolean instead of string.
And you can just do "isAuthinticated() ? a : b"

Reply:
Localstorage allows for storage of strings only, so even if you store boolean true/false you'll get stringified 'true'/false'. I'll parse it in isAuthenticated method and we can implement it the way you suggest.